### ISSUE TITLE
a tutorial for internal messages 

### ISSUE THREAD

**Issue by fdch** [pure-data/pure-data#651]
There has been some improvements and discussions on dynamic patching lately and I would like to share a personal guide on how I managed to work with internal messages a couple of years ago, partly using existing information from puredata.info and from the `g_*.c` files. Perhaps this could be useful to tidy up with better instructions on what to do and what not do do. 

https://github.com/fdch/fd_lib/tree/master/tutorials

> I think it would be good to have some sort of guidelines help patch for this in the manual. 

I'm up for that. I've been actually working on something like this in https://github.com/porres/Live-Electronic-Music-Tutorial - but it's not icluded yet, will be in the next version!

_Originally posted by @porres in https://github.com/pure-data/pure-data/issues/623#issuecomment-491453716_

**Comment by porres** [comment]
yup, I'm up for that :) and also for an external managing tutorial

**Comment by jmmmp** [comment]
There are some extensive lists in pd-extended\doc\manuals\pd-msg, which could be adapted for new documentation. Probably these will be outdated.

**Comment by porres** [comment]
@fdch, please open a new issue on https://github.com/pure-data/pddp and close this one, let's take it there.
